lighthouse-core/audits/audit.js

```diff
@@ -27,6 +27,16 @@ class Audit {
   }
 
   /**
+   * @return {{Numeric: string, Binary: string}}
+   */
+  static get ScoringModes() {
+    return {
+      Numeric: 'numeric',
```

Comment:
should be `NUMERIC` and `BINARY` or `numeric` and `binary` IMO. also `SCORING_MODES`?